crates/bevy_text/src/pipeline.rs

```diff
@@ -228,7 +137,7 @@ impl TextPipeline {
         );
 
         // TODO: other shaping methods?
-        buffer.shape_until_scroll(font_system);
+        buffer.set_rich_text(font_system, spans, Shaping::Basic);
```

Comment:
I think we probably ought to default to `Shaping::Advanced` (which uses rustybuzz) rather than `Shaping::Basic` which is quite naive:

```rust
/// The shaping strategy of some text.
#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
pub enum Shaping {
    /// Basic shaping with no font fallback.
    ///
    /// This shaping strategy is very cheap, but it will not display complex
    /// scripts properly nor try to find missing glyphs in your system fonts.
    ///
    /// You should use this strategy when you have complete control of the text
    /// and the font you are displaying in your application.
    #[cfg(feature = "swash")]
    Basic,
    /// Advanced text shaping and font fallback.
    ///
    /// You will need to enable this strategy if the text contains a complex
    /// script, the font used needs it, and/or multiple fonts in your system
    /// may be needed to display all of the glyphs.
    Advanced,
}
```

Reply:
For future readers: we decided against this for now to substantial performance costs.